ckanext/dhis2harvester/fanstatic/throbber.js

```diff
@@ -0,0 +1,17 @@
+// Enable JavaScript's strict mode. Strict mode catches some common
+// programming errors and throws exceptions, prevents some unsafe actions from
+// being taken, and disables some confusing and bad JavaScript features.
```

Comment:
I think we use "strict" everywhere and there's no need to document how it works in the comments.